jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -125,7 +125,7 @@ def test_gls_vs_ols_two_ints_ols():
     np.testing.assert_allclose(slopes_gls[0].data[50, 50], 150.0, 1e-6)
 
 
-@pytest.mark.skip(reason="Jenkins environment does not correctly handle multi-processing.")
+# @pytest.mark.skip(reason="Jenkins environment does not correctly handle multi-processing.")
```

Comment:
So does the Jenkins environment now handle multi-processing?

Reply:
I don't think so.  These tests should work in github actions.

Reply:
So if we enable them to run in CI/GA, will they cause a problem when they get run by our nightly regression tests in Jenkins?

Reply:
I don't know if they effect nightly regression tests.  I will find out.  The problem is that since Jenkins didn't actually have multiple processors, these tests weren't actually testing multiprocessing.  These tests passed, but ran on a single processor, giving the false impression multiprocessing got tested and passed.

Reply:
These were originally written by @mwregan2 and were skipped for CI because TravisCI couldn't run them in multiprocessing mode.  Now that we're doing Github Actions, we have 2 cores available, so as long as we use "all" cores, then this will be testing what we want it to test.  On the flip side, when run on the 30-core machine we run our nightly regression tests on under Jenkins, it might also split them over 30 cores.  Which again should be fine, and they should pass there too.